docs/docs/usage/hooks.md

```diff
@@ -0,0 +1,234 @@
+# Lifecycle and hooks
+
+As any Python deliverable, your project will go through the different phases
+of a Python project lifecycle and PDM provides commands to perform the expected tasks for those phases.
+
+It also provides hooks attached to these steps allowing for:
+
+- plugins to listen to the [signals](../plugin/reference.md#signals) of the same name.
+- developpers to define custom scripts with the same name.
+
+The built-in commands are currently splitted in 3 groups:
+
+- the [initialization phase](#initialization)
+- the [dependencies management](#dependencies-managment).
+- the [publication phase](#publication).
+
+You will most probably need to perform some recurrent tasks between th installation and publication phases (housekeeping, linting, testing, ...)
+this is why PDM lets you define your own tasks/phases using [user scripts](#user-scripts).
+
+To provides full flexibility, PDM allows to [skip some hooks and tasks](#skipping) on demand.
+
+## Initialization
+
+The intialization phase should occurs only once in a project lifetime by running on of the following commands:
+
+- `pdm new` to start a project from scratch
+- `pdm init` to initialize an existing project (prompt to fill the `pyproject.toml` file)
+
+They trigger the following hooks:
+
+- [`post_init`](../plugin/reference.md#pdm.signals.post_init)
```

Comment:
You can take advantage of the autorefs MkDocs plugin (which is used by mkdocstrings) and write ``[`post_init`][pdm.signals.post_init]``. The advantage is you don't have to specify the page, so the link will work even if the structure of the docs changes.

Reply:
Way better 👌🏼